sdks/java/core/src/test/java/org/apache/beam/sdk/options/PipelineOptionsFactoryTest.java

```diff
@@ -1790,4 +1795,35 @@ public void serialize(JacksonIncompatible jacksonIncompatible, JsonGenerator jso
     }
   }
 
+  /** An options class to for the following test, it will be loaded in its own classloader. */
```

Comment:
`An options class to for the following test, it will be loaded in its own classloader.` -> `Used to test that the thread context class loader is used when creating proxies.`